packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedback.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { ButtonIcon, ButtonPrimary, Flex, Link, Text } from 'design';
+import Validation from 'shared/components/Validation';
+import { Close } from 'design/Icon';
+import { ShareFeedbackForm } from './ShareFeedbackForm';
+import { ShareFeedbackFormValues } from './types';
+
+interface ShareFeedbackProps {
+  onClose(): void;
+}
+
+export function ShareFeedback(props: ShareFeedbackProps) {
+  const [formValues, setFormValues] = useState<ShareFeedbackFormValues>({
+    feedback: '',
+    email: '',
+    newsletterEnabled: false,
+    salesContactEnabled: false,
+  });
+
+  return (
+    <Flex bg="primary.main" p={3} borderRadius={3}>
+      <Validation>
+        {({ validator }) => (
+          <Flex
+            flexDirection="column"
+            as="form"
+            onSubmit={e => {
+              e.preventDefault();
+              if (validator.validate()) {
+                console.log('submit', formValues); //TODO (gzdunek): connect to a real service
```

Comment:
I was thinking how we could make it so that we're able to still execute this code in development.

Perhaps if the app is not packaged, we could just change the endpoint to which it sends the request?

I did a bit of googling and found that [Postman has its own echo service](https://documenter.getpostman.com/view/5025623/SWTG5aqV#0cc70253-566a-4fc8-b311-2bda4d955b84).

This would mean that in development we could just point our code to `https://postman-echo.com/post` and it should still work.

Reply:
This is a great idea, I use that service in the next PR when I will connect to the real backend.